src/frontends/pytorch/src/op/adaptive_max_pool2d.cpp

```diff
@@ -14,13 +14,8 @@ namespace op {
 OutputVector translate_adaptive_max_pool2d(NodeContext& context) {
     auto x = context.get_input(0);
     auto y = context.get_input(1);
-    auto adaptive_max_pool = context.mark_node(std::make_shared<opset8::AdaptiveMaxPool>(x, y));
-    auto return_indices = context.const_input<bool>(2);
-    OutputVector res{adaptive_max_pool->output(0)};
-    if (return_indices) {
-        res.push_back(adaptive_max_pool->output(1));
-    }
-    return res;
+    auto adaptive_max_pool = context.mark_node(std::make_shared<opset8::AdaptiveMaxPool>(x, y, ov::element::i32));
+    return {adaptive_max_pool->output(0), adaptive_max_pool->output(1)};
```

Comment:
Please explain what's wrong with the original code here. TS always have two destinations for this op and we are failing in case when `return_indices` is `False`, aren't we?

Reply:
On the TS side not matter what is the value of `return_indices` max pool operation always return 2 outputs. And then either only `outputs` (first tensor) is returned in the next stage as in the following graph
![Zrzut ekranu (78)](https://user-images.githubusercontent.com/72454773/209130502-0e986afa-5e9f-406b-8063-cdb3abfb3d9a.png)
On the other had if `return_indices` was true then additional `TupleConstruct` node is being added to the graph as follows
![Zrzut ekranu (86)](https://user-images.githubusercontent.com/72454773/209130724-4e3619bd-35ac-444a-8efa-e8b2e5d4740c.png)
So in general no matter of the value of `return_indices` we should always return 2 outputs.

Reply:
On the OV side if we return 2 outputs and `return_indices=False` then in the graph only first output of the `max_pool` is being passed to the `Result` node.
![Zrzut ekranu (80)](https://user-images.githubusercontent.com/72454773/209131227-2f0919a0-c5f1-413c-8ed8-544f395c633f.png)

Reply:
So in the end it works as expected

Reply:
OK, indeed. I cannot even see this bool input 2 in that TS code. So it is always true by default? OK, anyways I agree with this modification.